src/Symfony/Component/Console/Helper/SymfonyQuestionHelper.php

```diff
@@ -89,16 +97,27 @@ protected function writePrompt(OutputInterface $output, Question $question)
                 $text = sprintf(' <info>%s</info> [<comment>%s</comment>]:', $text, $default);
         }
 
-        $output->writeln($text);
+        $output->write($text);
 
         if ($question instanceof ChoiceQuestion) {
             $width = max(array_map('strlen', array_keys($question->getChoices())));
 
             foreach ($question->getChoices() as $key => $value) {
-                $output->writeln(sprintf("  [<comment>%-${width}s</comment>] %s", $key, $value));
+                $output->writeLn('');
+                $output->write(sprintf("  [<comment>%-${width}s</comment>] %s", $key, $value));
             }
         }
+    }
 
+    /**
+     * {@inheritdoc}
+     */
+    protected function prependResponse(OutputInterface $output, Question $question)
+    {
+        $output->writeLn('');
```

Comment:
I'm not sure this is needed. As we use `enter` to validate our answer it should already be there right?